models/bgrid_solo/readme.rst

```diff
@@ -27,10 +27,32 @@ model time step to maintain stability for larger model grids. The model state
 variables are the gridded surface pressure, temperature, and u and v wind
 components.
 
-The ``bgrid_solo`` directory has a ``work/workshop_setup.csh`` script that compiles 
-and runs an example. This example is intended to demonstrate that the same
-process used for a low-order model may be used for a much more 
-complex model and generates output for state-space or observation-space diagnostics. 
+The ``bgrid_solo`` directory has a ``work/workshop_setup.sh`` script that compiles
+and runs an example, producing a bgrid_solo netcdf file and perturbing it to get an
+ensemble. This example is intended to demonstrate that the same process used for a
+low-order model may be used for a much more complex model and generates output for
+state-space or observation-space diagnostics.
+
+Alternatively, the steps in the ``work/workshop_setup.sh`` script can be executed
+indivudally on the command line. These steps are as follows:
+
+1.  | ``./quickbuild.sh`` (or ``quickbuild.sh nompi`` if you are not building with mpi)
+    | This will create perfect_input.nc from the already available file
+      ``models/bgrid_solo/work/perfect_input.cdl``
+
+2.  | ``./perfect_model_obs``
+    | Creates a synthetic observation sequence from a hindcast model
+
+3.  | ``cp perfect_output.nc filter_input.nc``
+    | Copies the output from perfect_model_obs program to the input file for filter
+
+4.  | In ``work/input.nml``, set ``perturb_from_single_instance = .true.`` in the
+      &filter_nml
+    | This setting allows filter to generate an ensemble of perturbed ensemble member
+      restart files
+
+5.  | ``./filter``
+    | Runs the assimilation program
```

Comment:
1-4 is a nice description of what the workshop_setup.sh script is doing. You are a bit loose with your language though. If you think it is necessary to add this new documentation, I think it is worth being precise - see the comments on the individual lines. 

It might be better to explain the 'why' of each step rather then explaining the command. e.g. why are you coping the perfect_output.nc to filter_input.nc?, why are you setting perturb_from_single instance?

Reply:
I agree with Marlee that the steps to run the pmo experiment should be in readme.   I recommend going somewhat further and moving most of the text
from the beginning of workshop_setup to readme; everything from
'create_obs_sequence' through 'yet more matlab scripts'.  It should be combined with what Marlee has already added to readme, and some of it 
could use a little more (or less) context.
If we want the focus to be on the workshop_setup, then Marlee's text is good.
If we'd rather have the focus be on the steps, then it could be rearranged
to list the steps, and then note that they are all combined in workshop_setup.

In the section about Matlab scripts, are the model-space scripts in the local matlab directory?
And the 'observation space' scripts are in $DART/diagnostics/matlab?
Should there be an instruction about adding these directories to the Matlab path?

In pursuit of readability, when there's a list of descriptions, 
I find it helpful start each item on a new line.  
So, if the section about filter output stays in workshop_setup:
# 'filter' results in three files (at least): 
# + preassim.nc - the state of all ensemble members prior to the assimilation 
# + analysis.nc - the state of all ensemble members after the assimilation, and 
# + obs_seq.final - the ensemble members' estimates of the observations.
(I made a few corrections in addition to changing the formatting.)
If it's moved to readme, then use rst syntax to make it a list.

Reply:
Thanks for the suggestions Helen and Kevin. I will make sure to incporporate them and tidy up the docs a bit further once we decide if we want to keep the workshop_setup.sh script.